src/main/resources/feedbackQuestionRubricResultStatsTemplate.html

```diff
@@ -1,10 +1,17 @@
 <div class="panel-body">
```

Comment:
As mentioned before, can add `resultStatistics` here.

Reply:
Would it make more sense to add a new class here, instead of shifting `resultStatistics` from above div to this one?

Reply:
Adding resultStatistics here would make it duplicate right? Causing other operations with resultStatistics to  be unstable. Should I just add a new class here? Maybe `rubricStatistics` ?

Reply:
`rubricStatistics` is better